parquet/src/main/java/org/apache/iceberg/parquet/Parquet.java

```diff
@@ -266,6 +272,47 @@ private WriteBuilder createContextFunc(
       return this;
     }
 
+    private <T> void setBloomFilterConfig(
+        Context context,
+        MessageType parquetSchema,
+        BiConsumer<String, Boolean> withBloomFilterEnabled,
+        BiConsumer<String, Double> withBloomFilterFPP) {
```

Comment:
Is it possible to use the same Name compat function we use in the parquet writer to get the compatible column names? Just seems a bit odd to look up the paths when we know how to generate them?

Reply:
For example couldn't we keep the old layout of the code and just use

https://github.com/apache/iceberg/blob/afda8be25652d44d9339a79c6797b6bf20c55bd6/core/src/main/java/org/apache/iceberg/avro/AvroSchemaUtil.java#L483
```java
        for (Map.Entry<String, String> entry : columnBloomFilterEnabled.entrySet()) {
          String colPath =  makeCompatibleName(entry.getKey());
          String bloomEnabled = entry.getValue();
          parquetWriteBuilder.withBloomFilterEnabled(colPath, Boolean.parseBoolean(bloomEnabled));
        }
```

Reply:
Feel free to tell me if this is a bad idea, I think your approach is fine as well since it does seem to be a better validation

Reply:
It's simpler to use `makeCompatibleName`. I have made the changes. Thanks!